src/App.jsx

```diff
@@ -14,70 +15,65 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [dish, setDish] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {dish
+          ? `${dish} is selected`
+          : `No goods selected`
+        }
+        {dish
+          && (
             <button
-              data-cy="AddButton"
+              data-cy="ClearButton"
               type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+              className="delete ml-3"
+              onClick={() => setDish('')}
+            />
+          )}
+      </h1>
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
-
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
+      <table className="table">
+        <tbody>
+          {goods.map(good => (
+            <tr
+              key={good.id}
+              data-cy="Good"
+              className={dish === good
+                ? 'has-background-success-light'
+                : ''
+              }
             >
-              +
-            </button>
-          </td>
+              <td>
+                <button
+                  data-cy={dish === good
+                    ? 'RemoveButton'
+                    : 'AddButton'
+                  }
+                  type="button"
+                  className={`button ${dish === good && 'is-info'}`}
+                  onClick={dish === good
+                    ? () => setDish('')
+                    : () => setDish(good)
```

Comment:
```suggestion
                  onClick={() => setDish(dish === good ? '' : good)}
```

let's simplify this part